<!DOCTYPE html>
<html>
<head>
<style> 
body {
background:rgb(242, 245, 243);
min-width:950px;

font-family: Verdana, Arial, SunSans-Regular, Sans-Serif;
font-size: 15px;
}
.item {
	float:left;	
	}
.item1 {
	float:left;	
	margin-left:8%;
	}
.item2 {
	float:right;
	margin-right:8%;	
	}
img {width:85%;height:85%;}
.clear{clear:both;}
a { text-decoration: none; }
a:hover {text-decoration: underline}
.div
{
width:60%;
margin-left:auto;
margin-right:auto;
background-color:white;
border:2px solid grey;
-moz-border-radius:18px;
-khtml-border-radius:18px;
-webkit-border-radius:18px;
border-radius:18px;
padding:5px;
position: relative;
box-shadow:4px 8px 20px rgba(0, 0, 0, 0.386719);
}
.header
{
font-size:22px;
}
.content
{
  width:97%;	
  margin-left:auto;
margin-right:auto;
}
.line{ clear:both; height:1px;margin:1em 0;overflow:hidden;border-bottom:1px solid white;}
#footer{margin-bottom:30px;text-align:center;font-size:0.825em;color:#333;}

table.hovertable {
	
	color:#333333;
	border-width: 1px;
	border-color: #999999;
	border-collapse: collapse;
}
table.hovertable th {
	background-color:#EBEBEB;
	border-width: 1px;
	padding: 8px;
	border-style: solid;
	border-color: #a9c6c9;
}
table.hovertable tr {
	background-color:#EBEBEB;
}
table.hovertable td {
	border-width: 1px;
	padding: 8px;
	border-style: solid;
	border-color: #a9c6c9;
}
</style>
</head>
<body>
<div class='div'>
<div class='header'>
	<br><b><center>Systems biology analyses unravel underlying substrates </center><br><center>between Parkinson's and Crohn's disease</center></b>
</div>
<div class='content'>
	<div class='item'>
  <br>
	<h3>Introduction</h3>
    <p>Recent studies have revealed that endogenous circRNAs could generate proteins and the circRNA-derived proteins play important roles in cellular responses to environmental stress and myogenesis, thereby increasing the complexity of transcriptome and proteome. To gain more insights into the novel layer of gene activity, comprehensive identification of circRNAs with protein-coding potential is the first step. Here, to facilitate the biologists to realize this, we present an integrated tool, CircPro. As an automated high-throughput data analysis pipeline, it is capable of detecting circRNAs, predicting their protein-coding potential and discovering junction reads from Ribo-Seq data.</p>
    <p>Based on total/poly(A)- RNA-Seq and Ribo-Seq data, CircPro enables users to discover circRNAs with protein-coding potential. CircPro is composed of three modules as below:</p>
    <p align="center"><a href="workflow_placeholder.jpg" target="_blank">
    <img src='workflow_placeholder.jpg' alt='work flow' />
  </a></p>
    <p>&bull;	Module 1: circRNA detection</p>
    <p>The total/poly(A)- RNA sequencing reads are mapped to reference genome using <a href="http://bio-bwa.sourceforge.net/">BWA-MEM</a>. The generated SAM alignment is used by CIRI2 for de novo detection of circRNAs. The gene annotation in GTF file format is used to classify the identified circRNAs based on exon boundaries.</p>
    <p>&bull;	Module 2: protein-coding potential score</p>
    <p>To calculate circRNA coding potential score, first, CircPro extracts circRNA sequences. For the exonic circRNAs, the introns are removed. Then, <a href="http://cpc.cbi.pku.edu.cn/">CPC</a> is used to assess the protein-coding potential of circRNAs. This procedure will generate the coding/noncoding classification, the coding potential score and the information of open reading frame (ORF).</p>
    <p>&bull;	Module 3: junction reads from Ribo-Seq</p>
    <p>First, the Ribo-Seq reads are processed by removing adaptors using <a href="http://hannonlab.cshl.edu/fastx_toolkit/index.html">FASTX-Toolkit</a>. Then, the reads are mapped to rRNA library, such as <a href="http://www.ensembl.org/index.html">Ensembl</a> and <a href="http://rfam.xfam.org/">RFam</a>, to remove the rRNAs. The cleaned reads are mapped to reference genome using <a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Bowtie2</a>. The unmapped reads extracted from the alignment SAM file using <a href="http://samtools.sourceforge.net/">SAMtools</a> are further mapped to the library of circRNA junction sites, which is constructed by extracting N nucleotides from both sides of the junction site (N is the length of Ribo-Seq reads). Finally, CircPro generates a final circRNA list including the information of genomic position, coding status, ORF length and junction reads from Ribo-Seq.</p>
    <p>If you use data posted here in your scientific research, please cite us:</p>
    <p>Yanshi Hu*, Bentao She, Zhounan Yin, Xinjian Yu, Wenyi Wu, Ming Chen*. <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx446" target="_blank">Systems biology analyses unravel underlying substrates between Parkinson's and Crohn's disease. <i><b>Brain</b></i> 2020; 33:15. doi:10.1093/bioinformatics/btx446.</a></p>
    </div>
    <div class="clear"></div>
    <div class='item1'>
    <h3>Downloads</h3>
	Three downloads are available here:<br/>
	<p>Package source: &nbsp <a class="software" href="circpro.tar.gz"><b>CircPro.tar.gz</b></a><br/>
	Example data: &nbsp <a href="testdata.tar.gz">testdata.tar.gz</a><br/>
	Manual file: &nbsp <a href="circplant_manual.pdf">manual.pdf</a></p>
    </div>

    <div class='item2'>
    <h3>Contact</h3>
    

		For comments or suggestions, please contact us:<br/>

<pre>
    Tel/Fax: +86 (0)571-88206122
    Yanshi Hu: <a href="mailto:yanshihu@zju.edu.cn">yanshihu@zju.edu.cn</a>
    Ming Chen: <a href="mailto:mchen@zju.edu.cn">mchen@zju.edu.cn</a>
</pre>
		<p></p>
    </div>


</div>
<div class="clear"></div>
</div>
<div class="line"></div>
<div id="footer">&copy Copyright 2020 - <a href="http://bis.zju.edu.cn/binfo/">Ming Chen's Lab</a>, Zhejiang University.  </div>
</body>
</html>
