<!DOCTYPE html>
<html lang="en-us" xmlns="http://www.w3.org/1999/xhtml">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.30.2" />
  <meta name="author" content="Yanshi HU">
  <meta name="description" content="Doctoral student">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="keywords" content="百度地图,百度地图API，百度地图自定义工具，百度地图所见即所得工具" />
  <meta name="description" content="百度地图API自定义地图，帮助用户在可视化操作下生成百度地图" />


  <link rel="stylesheet" href="/css/highlight.min.css">
  <link rel="stylesheet" href="/css/bootstrap.min.css">
  <link rel="stylesheet" href="/css/font-awesome.min.css">
  <link rel="stylesheet" href="/css/academicons.min.css">
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  
<!--
  <link rel="alternate" href="https://yanshihu.github.io/index.xml" type="application/rss+xml" title="Yanshi HU">
  <link rel="feed" href="https://yanshihu.github.io/index.xml" type="application/rss+xml" title="Yanshi HU">
-->
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="https://yanshihu.github.io/">

  <script type="text/javascript" src="http://api.map.baidu.com/api?v=2.0&ak=TBGiZnGdK994dgWaciPO2o46BzBTHji4"></script>
  

  <title>胡言石</title>


</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/cn.html">胡言石</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        <li class="nav-item">
          <a href="/cn.html#about_cn" data-target="#about_cn">
            
            <span>主页</span>
          </a>
        </li>


        <li class="nav-item">
          <a href="/cn.html#publications_cn" data-target="#publications_cn">
            
            <span> 出版物</span>
          </a>
        </li>

		
        <li class="nav-item">
          <a href="/cn.html#talks_cn" data-target="#talks_cn">
            
            <span>学术报告</span>
          </a>
        </li>
		
		
        <li class="nav-item">
          <a href="/cn.html#posters_cn" data-target="#posters_cn">
            
            <span>海报</span>
          </a>
        </li>


        <li class="nav-item">
          <a href="/cn.html#projects_cn" data-target="#projects_cn">
            
            <span>科研项目</span>
          </a>
        </li>


        <li class="nav-item">
          <a href="/cn.html#working_cn" data-target="#working_cn">
            
            <span>工作经历</span>
          </a>
        </li>

<!--
        <li class="nav-item">
          <a href="/CV/Yanshi Hu_CV.pdf" target="_blank">
            
            <span>CV</span>
          </a>
        </li>
-->

        <li class="nav-item">
          <a href="/cn.html#contact_cn" data-target="#contact_cn">
            
            <span>联系方式</span>
          </a>
        </li>

  

        <li class="nav-item">
          <a href="/">
            
            <span>英文版</span>
          </a>
        </li>

        
        

        
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>



  
  
  <section id="about_cn" class="home-section">
    <div class="container">
      



<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-address">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('https://yanshihu.github.io/img/xhr.jpg');">
      </div>
      <meta itemprop="image" content="https://yanshihu.github.io/img/xhr.jpg">
      

      <div class="portrait-title">
        <h2 itemprop="name">胡言石</h2>
        <h3 itemprop="jobTitle">博士研究生</h3>
        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          <a href="http://www.zju.edu.cn/" target="_blank" itemprop="url">
            <span itemprop="name">浙江大学</span>
          </a>
        </h3>
        
      </div>

      <link itemprop="url" href="https://yanshihu.github.io/">

      <ul class="social-icon" aria-hidden="true">
        
        
        <li>
          <a itemprop="sameAs" href="//twitter.com/Yanshi_Rocky_Hu" target="_blank">
            <i class="fa fa-twitter big-icon"></i>
          </a>
        </li>
        
<!--        
        <li>
          <a itemprop="sameAs" href="//weibo.com/u/1880787543" target="_blank">
            <i class="fa fa-weibo big-icon"></i>
          </a>
        </li>
-->        
        
        <li>
          <a itemprop="sameAs" href="//yanshihu.github.io/img/wechat_QR.jpg" target="_blank">
            <i class="fa fa-wechat big-icon"></i>
          </a>
        </li>        

        
        <li>
          <a itemprop="sameAs" href="//github.com/YanshiHu" target="_blank">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="//scholar.google.com/citations?user=MUMFezEaXZ0C&hl=en" target="_blank">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
<!--       
        <li>
          <a itemprop="sameAs" href="//www.ncbi.nlm.nih.gov/pubmed/?term=Guangchuang&#43;Yu[Author&#43;-&#43;Full]" target="_blank">
            <i class="ai ai-pubmed big-icon"></i>
          </a>
        </li>
-->        
        <li>
          <a itemprop="sameAs" href="//www.researchgate.net/profile/Yanshi_Hu" target="_blank">
            <i class="ai ai-researchgate big-icon"></i>
          </a>
        </li>

        
        <li>
          <a itemprop="sameAs" href="//orcid.org/0000-0003-3219-3778" target="_blank">
            <i class="ai ai-orcid big-icon"></i>
          </a>
        </li>
        
<!--        
        <li>
          <a itemprop="sameAs" href="//impactstory.org/u/0000-0002-6485-8781" target="_blank">
            <i class="ai ai-impactstory big-icon"></i>
          </a>
        </li>
-->        
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">

    

<h1 id="biography">个人简介</h1>

<p>我目前是浙江大学生命科学学院生物信息学系的博士研究生，拥有广泛的研究兴趣，包括：生物信息学、计算系统生物学及基于生物网络的常见神经退行性疾病——阿尔兹海默病（AD）和帕金森病（PD）——分子机制研究。</p>

    <div class="row">

      
      <div class="col-sm-5">
        <h3>研究兴趣</h3>
        <ul class="ul-interests">
          
          <li>生物信息学</li>
          
          <li>计算系统生物学</li>
          
          <li>神经退行性疾病机制研究</li>
  
        </ul>
      </div>
      

      
      <div class="col-sm-7">
        <h3>教育经历</h3>
        <ul class="ul-edu fa-ul">
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">生物信息学博士, 2017至今（在读）</p>
              <p class="institution">浙江大学</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">生物医学工程硕士, 2016</p>
              <p class="institution">天津医科大学</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">生物医学工程学士, 2013</p>
              <p class="institution">泰山医学院/山东第一医科大学（筹）</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>

</div>

    </div>
  </section>

  
  
  <section id="publications_cn" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>近期发表文章</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://alzres.biomedcentral.com/articles/10.1186/s13195-017-0252-z" target="_blank">
        <img src="/img/Alzgset.jpg" class="pub-banner"
             itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://alzres.biomedcentral.com/articles/10.1186/s13195-017-0252-z" target="_blank" itemprop="url">Analyzing the genes related to Alzheimer’s disease via a network and pathway-based approach</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Our understanding of the molecular mechanisms underlying Alzheimer’s disease (AD) remains incomplete. In this study, we collected genes potentially associated with AD by screening publications on genetic association studies deposited in PubMed. The major biological themes linked with these genes were then revealed by function and biochemical pathway enrichment analysis, and the relation between the pathways was explored by pathway crosstalk analysis. Furthermore, the network features of these AD-related genes were analyzed in the context of human interactome and an AD-specific network was inferred using the Steiner minimal tree algorithm. We compiled 430 human genes reported to be associated with AD from 823 publications. Biological theme analysis indicated that the biological processes and biochemical pathways related to neurodevelopment, metabolism, cell growth and/or survival, and immunology were enriched in these genes. Pathway crosstalk analysis then revealed that the significantly enriched pathways could be grouped into three interlinked modules—neuronal and metabolic module, cell growth/survival and neuroendocrine pathway module, and immune response-related module—indicating an AD-specific immune-endocrine-neuronal regulatory network. Furthermore, an AD-specific protein network was inferred and novel genes potentially associated with AD were identified.By means of network and pathway-based methodology, we explored the pathogenetic mechanism underlying AD at a systems biology level. Results from our work could provide valuable clues for understanding the molecular mechanism underlying AD. In addition, the framework proposed in this study could be used to investigate the pathological molecular network and genes relevant to other complex diseases or phenotypes.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        <strong>YS Hu</strong>, J Xin, Y Hu, L Zhang<sup>*</sup>, J Wang<sup>*</sup>
        
      </div>

      <div class="pub-publication">
        
        <em>Alzheimer's Research & Therapy</em>
        
      </div>
	  
	  <div class="pub-links">
        
<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pubmed/28446202" target="_blank">
  PubMed
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5406904" target="_blank">
  PMC
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://www.doi.org/10.1186/s13195-017-0252-z" target="_blank">
  DOI
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://biomedcentral.altmetric.com/details/19645234" target="_blank">
  Altmetric
</a>

<a class="btn btn-primary btn-outline btn-xs" href="/PDF/s13195-017-0252-z.pdf" target="_blank">
  PDF
</a>

<!--
<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/presentation/2016-ggtree-chinar">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ggtree">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ggtree/">
  Project
</a>
-->


      </div>
    </div>
	</div>
</div>


    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    <div class="col-md-12">
      <a href="https://sci-hub.tw/10.1007/s12035-016-9998-8" target="_blank">
        <img src="/img/PDgset.gif" class="pub-banner"
             itemprop="image">
      </a>
    </div>  
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://sci-hub.tw/10.1007/s12035-016-9998-8" target="_blank" itemprop="url">Network and Pathway-Based Analyses of Genes Associated with Parkinson's Disease</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Parkinson's disease (PD) is a major neurodegenerative disease influenced by both genetic and environmental factors. Although previous studies have provided insights into the significant impacts of genetic factors on PD, the molecular mechanism underlying PD remains largely unclear. Under such situation, a comprehensive analysis focusing on biological function and interactions of PD-related genes will provide us valuable information to understand the pathogenesis of PD. In the current study, by reviewing the literatures deposited in PUBMED, we identified 242 genes genetically associated with PD, referred to as PD-related genes gene set (PDgset). Functional analysis revealed that biological processes and biochemical pathways related to neurodevelopment, metabolism, and immune system were enriched in PDgset. Then, pathway crosstalk analysis indicated that the enriched pathways could be grouped into two modules, with one module consisted of pathways mainly involved in neuronal signaling and another in immune response. Further, based on a global human interactome, we found that PDgset tended to have more moderate degree compared with cancer-related genes. Moreover, PD-specific molecular network was inferred using Steiner minimal tree algorithm and some potential related genes associated with PD were identified. In summary, by using network- and pathway-based methods to explore pathogenetic mechanism underlying PD, results from our work may have important implications for understanding the molecular mechanism underlying PD. Also, the framework proposed in our current work can be used to infer pathological molecular network and genes related to a specific disease.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        <strong>YS Hu</strong>, Z Pan, Y Hu, L Zhang<sup>*</sup>, J Wang<sup>*</sup>
        
      </div>

      <div class="pub-publication">
        
        <em>Molecular Neurobiology</em>
        
      </div>

	  
	  <div class="pub-links">
        
<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pubmed/27349437" target="_blank">
  PubMed
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://www.doi.org/10.1007/s12035-016-9998-8" target="_blank">
  DOI
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://springeropen.altmetric.com/details/9126069" target="_blank">
  Altmetric
</a>

      </div>	  
    </div>	  
    </div>
	</div>

	
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">
	
    <div class="col-md-12">
      <a href="https://www.sciencedirect.com/science/article/pii/S1046202317300531" target="_blank">
        <img src="/img/Methods_Pathway_Relationship.jpg" class="pub-banner"
             itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://www.sciencedirect.com/science/article/pii/S1046202317300531" target="_blank" itemprop="url">Detecting pathway relationship in the context of human protein-protein interaction network and its application to Parkinson’s disease</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        In human physiological conditions like complex diseases, a large number of genes/proteins, as well as their interactions, are involved. Thus, detecting the biochemical pathways enriched in these genes/proteins and identifying the pathway relationships is critical to understand the molecular mechanisms underlying a disease and can also be valuable in selecting the potential molecular targets for further exploration. In this study, we proposed a method to measure the relationship between pathways based on their distribution in the human PPI network. By representing each pathway as a gene module in the PPI network, a distance was calculated to measure the closeness of two pathways. For the pathways in the KEGG database, a total of 2143 pathway pairs with close connections were identified. Additional evaluations indicated the pathway relationship built via such approach was consistent with available evidence. Further, based on the genes and pathways potentially associated with the pathogenesis of Parkinson’s disease (PD), we analyzed the pathway relationship and identified the major pathways related to this disorder via the new method. Also, by analyzing the pathway interaction network constructed by the identified major pathways, we explored the potential pathway targets that may be important in the etiology and development of PD. In summary, we proposed an approach to measure the relationship between pathways, which could provide a more systematic profile on pathways involved in a phenotype, and may also help to improve the result of pathway enrichment analysis.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Y Hu, Y Yang, Z Fang, <strong>YS Hu</strong>, L Zhang<sup>*</sup>, J Wang<sup>*</sup>
        
      </div>

      <div class="pub-publication">
        
        <em>Methods</em>
        
      </div>
	  
	  <div class="pub-links">
        
<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pubmed/28790017" target="_blank">
  PubMed
</a>
<!--
<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5406904" target="_blank">
  PMC
</a>
-->
<a class="btn btn-primary btn-outline btn-xs" href="https://www.doi.org/10.1016/j.ymeth.2017.08.001" target="_blank">
  DOI
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://plu.mx/plum/a/?doi=10.1016%2Fj.ymeth.2017.08.001&theme=plum-sciencedirect-theme&hideUsage=true&display-tab=summary-content" target="_blank">
  PlumX
</a>

<a class="btn btn-primary btn-outline btn-xs" href="/PDF/1-s2.0-S1046202317300531-main.pdf" target="_blank">
  PDF
</a>

<!--
<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/presentation/2016-ggtree-chinar">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ggtree">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ggtree/">
  Project
</a>
-->


      </div>
	  
    </div>
    </div>
	</div>

	
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">
  
    <div class="col-md-12">
      <a href="https://academic.oup.com/database/article/doi/10.1093/database/bax097/4774645" target="_blank">
        <img src="/img/Database_GRONS.jpg" class="pub-banner"
             itemprop="image">
      </a>
    </div>	
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://academic.oup.com/database/article/doi/10.1093/database/bax097/4774645" target="_blank" itemprop="url">GRONS: a comprehensive genetic resource of nicotine and smoking</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Nicotine, the primary psychoactive component in tobacco, can exert a broad impact on both the central and peripheral nervous systems. During the past years, a tremendous amount of efforts has been put to exploring the molecular mechanisms underlying tobacco smoking related behaviors and diseases, and many susceptibility genes have been identified via various genomic approaches. For many human complex diseases, there is a trend towards collecting and integrating the data from genetic studies and the biological information related to them into a comprehensive resource for further investigation, but we have not found such an effort for nicotine addiction or smoking-related phenotypes yet. To collect, curate, and integrate cross-platform genetic data so as to make them interpretable and easily accessible, we developed Genetic Resources Of Nicotine and Smoking (GRONS), a comprehensive database for genes related to biological response to nicotine exposure, tobacco smoking related behaviors or diseases. GRONS deposits genes from nicotine addiction studies in the following four categories, i.e. association study, genome-wide linkage scan, expression analysis on genes/proteins via high-throughput technologies, as well as single gene/protein-based experimental studies via literature search. Moreover, GRONS not only provides tools for data browse, search and graphical presentation of gene prioritization, but also presents the results from comprehensive bioinformatics analyses for the prioritized genes associated with nicotine addiction. With more and more genetic data and analysis tools integrated, GRONS will become a useful resource for studies focusing on nicotine addiction or tobacco smoking. Database URL: <a href="http://bioinfo.tmu.edu.cn/GRONS/"  target="_blank">bioinfo.tmu.edu.cn/GRONS</a>
      </div>
	  <p>

      <div class="pub-authors" itemprop="author">
        
        Z Fang, Y Yang, <strong>YS Hu</strong>, Ming D Li<sup>*</sup>, J Wang<sup>*</sup>
        
      </div>

      <div class="pub-publication">
        
        <em>Database</em>
        
      </div>
	  	  
	  <div class="pub-links">
<!--        
<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pubmed/28446202" target="_blank">
  PubMed
</a>
-->
<a class="btn btn-primary btn-outline btn-xs" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5750854" target="_blank">
  PMC
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://www.doi.org/10.1093/database/bax097" target="_blank">
  DOI
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://oxfordjournals.altmetric.com/details/30904936" target="_blank">
  Altmetric
</a>

<a class="btn btn-primary btn-outline btn-xs" href="/PDF/bax097.pdf" target="_blank">
  PDF
</a>

<!--
<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/presentation/2016-ggtree-chinar">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ggtree">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ggtree/">
  Project
</a>
-->


      </div>	  
    </div>	
	
    </div>
	</div>	
<!--
      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/ggtree/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://onlinelibrary.wiley.com/doi/10.1111/2041-210X.12628/abstract">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/presentation/2016-ggtree-chinar">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ggtree">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ggtree/">
  Project
</a>



      </div>
-->



<!--    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://guangchuangyu.github.io/publication/chipseeker/" itemprop="url">ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Summary: ChIPseeker is an R package for annotating ChIP-seq data analysis. It supports annotating ChIP peaks and provides functions to visualize ChIP peaks coverage over chromosomes and profiles of peaks binding to TSS regions. Comparison of ChIP peak profiles and annotation are also supported. Moreover, it supports evaluating significant overlap among ChIP-seq datasets. Currently, ChIPseeker contains 15 000 bed file information from GEO database. These datasets can be downloaded and compare with user&#39;s own data to explore significant overlap datasets for inferring co-regulation or transcription factor complex for further investigation.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        <strong>G Yu</strong>, LG Wang, QY He<sup>*</sup>
        
      </div>

      <div class="pub-publication">
        
        <em>Bioinformatics</em>
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/chipseeker/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/31/14/2382">
  PDF
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ChIPseeker">
  Code
</a>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ChIPseeker/">
  Project
</a>



      </div>

    </div>
  </div>
</div>
-->
    
    

  </div>
</div>

    </div>
  </section>

  
<!--  
  <section id="publications" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Recent Publications</h1>
    
    
    <p class="view-all">
      <a href="/publication/">
        More Publications
        <i class="fa fa-angle-double-right"></i>
      </a>
    </p>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    <ul class="fa-ul">
      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/ggtree/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://onlinelibrary.wiley.com/doi/10.1111/2041-210X.12628/abstract">
  PDF
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/presentation/2016-ggtree-chinar">
  Slides
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Video
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ggtree">
  Code
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/">
  Dataset
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ggtree/">
  Project
</a>


</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">ReactomePA: an R/Bioconductor package for reactome pathway analysis and visualization</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/reactomepa/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://pubs.rsc.org/en/content/articlepdf/2016/mb/c5mb00663e">
  PDF
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ReactomePA">
  Code
</a>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ReactomePA/">
  Project
</a>


</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparison and visualization</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/chipseeker/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/31/14/2382">
  PDF
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/ChIPseeker">
  Code
</a>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/ChIPseeker/">
  Project
</a>


</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">DOSE: an R/Bioconductor package for disease ontology semantic and enrichment analysis</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/dose/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/31/4/608">
  PDF
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/DOSE">
  Code
</a>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/DOSE/">
  Project
</a>


</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Putative cobalt- and nickel-binding proteins and motifs in Streptococcus pneumoniae</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/2013_metallomics/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://pubs.rsc.org/en/content/articlelanding/2013/mt/c3mt00126a">
  PDF
</a>







</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">A novel andrographolide derivative AL-1 exerts its cytotoxicity on K562 cells through a ROS-dependent mechanism</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/2013_proteomics/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200273/abstract">
  PDF
</a>







</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">LXtoo: an integrated live Linux distribution for the bioinformatics community</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/lxtoo/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://www.biomedcentral.com/1756-0500/5/360/abstract">
  PDF
</a>







</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">clusterProfiler: an R Package for Comparing Biological Themes Among Gene Clusters</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/clusterprofiler/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://online.liebertpub.com/doi/abs/10.1089/omi.2011.0118">
  PDF
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/GuangchuangYu/clusterProfiler">
  Code
</a>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/clusterProfiler/">
  Project
</a>


</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">Functional similarity analysis of human virus-encoded miRNAs</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/2011_virus_mirna/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://www.jclinbioinformatics.com/content/1/1/15">
  PDF
</a>







</p>
</li>

      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-file-text-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name">A new method for measuring functional similarity of microRNAs</span>
  <p>



<a class="btn btn-primary btn-outline btn-xs" href="https://guangchuangyu.github.io/publication/2011_mirna/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="http://www.jiomics.com/index.php/jio/article/view/21">
  PDF
</a>







</p>
</li>

      
    </ul>
    

  </div>
</div>

    </div>
  </section>
-->
  
<!--  
  <section id="talks_cn" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Recent &amp; Upcoming Talks</h1>
    
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    <ul class="fa-ul">
      
      <li itemscope itemtype="http://schema.org/CreativeWork">
  <i class="fa-li fa fa-comment-o pub-icon" aria-hidden="true"></i>
  <span itemprop="name"><a href="https://guangchuangyu.github.io/talk/example-talk/">Example Talk</a></span>
  <p>
    Sun, Jan 1, 2017,
    
            Hugo Academic Theme Conference
    
  </p>
</li>

      
    </ul>
    

  </div>
</div>

    </div>
  </section>
-->
  
  
  <section id="posters_cn" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">

    <h1>近期海报展示</h1>
    
    
    <p class="view-all">
      <a href="/poster/poster.html" target="_blank">
        更多海报展示
        <i class="fa fa-angle-double-right"></i>
      </a>
    </p>
    

  </div>
<!--  
  <div class="col-xs-12 col-md-8">

    

    
      
        
        


<div class="article-list-item" itemscope itemprop="blogPost">
  
  
  <h3 class="article-title" itemprop="name">
    <a href="https://guangchuangyu.github.io/2018/04/rename-phylogeny-tip-labels-in-treeio/" itemprop="url">rename phylogeny tip labels in treeio</a>
  </h3>
  

<div class="article-metadata">

  <span class="article-date">
    <time datetime="2018-04-19 11:00:08 &#43;0800 HKT" itemprop="datePublished">
      Thu, Apr 19, 2018
    </time>
  </span>

  
  
  
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/evolution">Evolution</a
    >, 
    
    <a href="/categories/visualization">Visualization</a
    >, 
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  
  
  
  <span class="article-tags">
    <i class="fa fa-tags"></i>
    
    <a href="/tags/ggtree">ggtree</a
    >, 
    
    <a href="/tags/treeio">treeio</a
    >
    
  </span>
  
  

  

</div>

  <div class="article-style" itemprop="articleBody">
    
    I don’t know whether ‘rename taxa’ is a common task or not. It seems not a good idea to rename taxa in Newick tree text, since it may introduce problems when mapping the original sequence alignment to the tree.
If you just want to show different or additional information when plotting the tree, it is fine and easy to do it using ggtree:
 require(treeio) ## Loading required package: treeio require(ggtree) ## Loading required package: ggtree ## ggtree v1.
    
  </div>
  <p class="read-more">
    <a href="https://guangchuangyu.github.io/2018/04/rename-phylogeny-tip-labels-in-treeio/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      
        
        


<div class="article-list-item" itemscope itemprop="blogPost">
  
  
  <h3 class="article-title" itemprop="name">
    <a href="https://guangchuangyu.github.io/2018/04/setting-ggplot2-background-with-ggbackground/" itemprop="url">setting ggplot2 background with ggbackground</a>
  </h3>
  

<div class="article-metadata">

  <span class="article-date">
    <time datetime="2018-04-18 21:00:00 &#43;0800 HKT" itemprop="datePublished">
      Wed, Apr 18, 2018
    </time>
  </span>

  
  
  
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/r">R</a
    >, 
    
    <a href="/categories/visualization">Visualization</a
    >
    
  </span>
  
  

  
  
  
  <span class="article-tags">
    <i class="fa fa-tags"></i>
    
    <a href="/tags/ggimage">ggimage</a
    >
    
  </span>
  
  

  

</div>

  <div class="article-style" itemprop="articleBody">
    
    ggimage 0.1.4 is available on CRAN.
This release introduces a new function called ggbackground for setting image background as ggplot canvas.
require(ggplot2) p &lt;- ggplot(iris) + aes(x = Sepal.Length, y = Sepal.Width, color=Species) + geom_point(size=5) + theme_classic() Suppose we have the above ggplot object, p, the only thing we need to do is passing the p with an image file name (local or remote) to ggbackground, as demonstrated below:
 require(ggimage) img = &quot;https://assets.
    
  </div>
  <p class="read-more">
    <a href="https://guangchuangyu.github.io/2018/04/setting-ggplot2-background-with-ggbackground/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      
        
        


<div class="article-list-item" itemscope itemprop="blogPost">
  
  
  <h3 class="article-title" itemprop="name">
    <a href="https://guangchuangyu.github.io/2018/03/annotating-phylogenetic-tree-with-images-using-ggtree-and-ggimage/" itemprop="url">Annotating phylogenetic tree with images using ggtree and ggimage</a>
  </h3>
  

<div class="article-metadata">

  <span class="article-date">
    <time datetime="2018-03-01 11:10:48 &#43;0800 HKT" itemprop="datePublished">
      Thu, Mar 1, 2018
    </time>
  </span>

  
  
  
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/visualization">Visualization</a
    >, 
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  
  
  
  <span class="article-tags">
    <i class="fa fa-tags"></i>
    
    <a href="/tags/ggtree">ggtree</a
    >, 
    
    <a href="/tags/ggimage">ggimage</a
    >, 
    
    <a href="/tags/tree">tree</a
    >
    
  </span>
  
  

  

</div>

  <div class="article-style" itemprop="articleBody">
    
    With ggimage, we are able to plot images using grammar of graphics. The layers defined in ggimage can be directly applied to ggtree to annotate phylogenetic tree using local/online image files.
ggtree seamlessly work with ggimage. The geom_tiplab and geom_nodelab can accept parameter of geom=&quot;image&quot; to parse taxa labels as image files and use them to “label” the taxa using images instead of text strings. Here are some examples for demonstration.
    
  </div>
  <p class="read-more">
    <a href="https://guangchuangyu.github.io/2018/03/annotating-phylogenetic-tree-with-images-using-ggtree-and-ggimage/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      
        
        


<div class="article-list-item" itemscope itemprop="blogPost">
  
  
  <h3 class="article-title" itemprop="name">
    <a href="https://guangchuangyu.github.io/2017/12/visualizing-reassortment-history-using-seqcombo/" itemprop="url">visualizing reassortment history using seqcombo</a>
  </h3>
  

<div class="article-metadata">

  <span class="article-date">
    <time datetime="2017-12-05 11:09:07 &#43;0800 HKT" itemprop="datePublished">
      Tue, Dec 5, 2017
    </time>
  </span>

  
  
  
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/genomics">Genomics</a
    >, 
    
    <a href="/categories/r">R</a
    >
    
  </span>
  
  

  
  
  
  <span class="article-tags">
    <i class="fa fa-tags"></i>
    
    <a href="/tags/seqcombo">seqcombo</a
    >, 
    
    <a href="/tags/influenza">influenza</a
    >
    
  </span>
  
  

  

</div>

  <div class="article-style" itemprop="articleBody">
    
    <p>Reassortment is an important strategy for influenza A viruses to
introduce a HA subtype that is new to human populations, which creates
the possibilities of pandemic.</p>

<p><img src="https://raw.githubusercontent.com/GuangchuangYu/seqcombo/master/vignettes/figures/srep25549-f2.jpg" alt="" /></p>

<p>A diagram showed above (Figure 2 of <code>doi:10.1038/srep25549</code>) is widely
used to illustrate the reassortment events. While such diagrams are
mostly manually draw and edit without software tool to automatically
generate. Here, I implemented the <code>hybrid_plot</code> function for producing
publication quality figure of reassortment events.</p>

<pre><code>library(tibble)
library(ggplot2)


n &lt;- 8

virus_info &lt;- tibble(
    id = 1:7,
    x = c(rep(1990, 4), rep(2000, 2), 2009),
    y = c(1,2,3,5, 1.5, 3, 4),
    segment_color = list(
        rep('purple', n),
        rep('red', n),
        rep('darkgreen', n),
        rep('lightgreen', n),
        c('darkgreen', 'darkgreen', 'red', 'darkgreen', 'red', 'purple', 'red', 'purple'),
        c('darkgreen', 'darkgreen', 'red', 'darkgreen', 'darkgreen', 'purple', 'red', 'purple'),
        c('darkgreen', 'lightgreen', 'lightgreen', 'darkgreen', 'darkgreen', 'purple', 'red', 'purple'))
)

flow_info &lt;- tibble(from = c(1,2,3,3,4,5,6),
                    to = c(5,5,5,6,7,6,7))
hybrid_plot(virus_info, flow_info)
</code></pre>

<p><img src="http://guangchuangyu.github.io/blog_images/Bioconductor/seqcombo/reassortment_files/figure-markdown_strict/unnamed-chunk-2-1.png" alt="" /></p>

<p></p>
    
  </div>
  <p class="read-more">
    <a href="https://guangchuangyu.github.io/2017/12/visualizing-reassortment-history-using-seqcombo/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      
        
        


<div class="article-list-item" itemscope itemprop="blogPost">
  
  
  <h3 class="article-title" itemprop="name">
    <a href="https://guangchuangyu.github.io/2017/11/bmc-journal-protects-paper-of-plagiarism/" itemprop="url">BMC journal protects paper of plagiarism</a>
  </h3>
  

<div class="article-metadata">

  <span class="article-date">
    <time datetime="2017-11-09 21:51:43 &#43;0800 HKT" itemprop="datePublished">
      Thu, Nov 9, 2017
    </time>
  </span>

  
  
  
  <span class="article-categories">
    <i class="fa fa-folder"></i>
    
    <a href="/categories/biology">Biology</a
    >, 
    
    <a href="/categories/research">Research</a
    >
    
  </span>
  
  

  
  
  
  <span class="article-tags">
    <i class="fa fa-tags"></i>
    
    <a href="/tags/bmc">BMC</a
    >, 
    
    <a href="/tags/plagiarism">plagiarism</a
    >, 
    
    <a href="/tags/gosemsim">GOSemSim</a
    >, 
    
    <a href="/tags/ppipre">ppiPre</a
    >
    
  </span>
  
  

  

</div>

  <div class="article-style" itemprop="articleBody">
    
    <p>My friend who doing his PhD study at Johns Hopkins just send me the link about <a href="https://retractionwatch.com/2017/11/07/17-johns-hopkins-researchers-resign-protest-ed-board-nature-journal/" target="_blank">a SR paper of plagiarism</a>. I have very similar experence of <a href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-S2-S8" target="_blank">a paper published on BMC Systems Biology</a>, which plagiarized <a href="https://www.ncbi.nlm.nih.gov/pubmed/20179076" target="_blank">my work</a> and the editor just decided to publish an erratum.</p>

<p>Deng <em>etc.</em> published an R package, ppiPre, that copied source code of my package, GOSemSim, and pretended that they developed these algorithms by themselves in their paper.</p>

<p>Here is the screenshot of the source code (left: ppiPre, right: GOSemSim).</p>

<p><img src="https://guangchuangyu.github.io/blog_images/screenshots/Screenshot 2017-07-17 19.43.51.png" alt="" /></p>

<p>You can find out more on <a href="http://guangchuangyu.github.io/2014/11/proper-use-of-gosemsim/" target="_blank">my blog post</a>.</p>

<p>As a developer of several open source software, I am glad that someone find my source code useful and happy if someone use my source code to make something better. But I am not happy if someone copies my source code by removing author information and changing function names to pretend the code was developed by himself. The situation is even worse in academic. Taking someone else&rsquo;s works and passing it off as one&rsquo;s own is definitely plagiarism and not allow in academic.</p>

<p></p>
    
  </div>
  <p class="read-more">
    <a href="https://guangchuangyu.github.io/2017/11/bmc-journal-protects-paper-of-plagiarism/" class="btn btn-primary btn-outline">
      CONTINUE READING
    </a>
  </p>
</div>

      
    
-->
  </div>
</div>

    </div>
  </section>

  
  
  <section id="projects_cn" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>科研项目</h1>
    
  </div>
<!--
  <div class="col-xs-12 col-md-8">
    

    <div class="project-toolbar">
      <div id="filters">
        <div class="btn-toolbar">
          <div class="btn-group">
          
            <a href="#" data-filter="*" class="btn btn-primary btn-large active">All</a>
          
            <a href="#" data-filter=".bioc" class="btn btn-primary btn-large">Bioconductor</a>
          
            <a href="#" data-filter=".cran" class="btn btn-primary btn-large">CRAN</a>
          
          </div>
        </div>
      </div>
    </div>

    

    <div id="container-projects" class="row isotope">

      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioc">
        <div class="card">
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/meshes" target="_blank">meshes</a></h4>
            <div class="card-desription">
              <p>This package provides functions for pathway analysis based on REACTOME pathway database. It implements enrichment analysis, gene set enrichment analysis and several functions for visualization.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item cran">
        <div class="card">
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/emojifont" target="_blank">emojifont</a></h4>
            <div class="card-desription">
              <p>emojifont is an implementation of using emoji font in both base and &lsquo;ggplot2&rsquo; graphics.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioc">
        <div class="card">
          
          <a href="https://guangchuangyu.github.io/ggtree" title="" class="card-image hover-overlay"
             target="_blank">
            <img src="/img/C2mxyBuUcAEt391.jpg" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/ggtree" target="_blank">ggtree</a></h4>
            <div class="card-desription">
              <p>The ggtree package extending the ggplot2 package. It based on grammar of graphics and takes all the good parts of ggplot2. ggtree is designed for not only viewing phylogenetic tree but also displaying annotation data on the tree.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioc">
        <div class="card">
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/ChIPseeker" target="_blank">ChIPseeker</a></h4>
            <div class="card-desription">
              <p>This package implements functions to retrieve the nearest genes around the peak, annotate genomic region of the peak, statstical methods for estimate the significance of overlap among ChIP peak data sets, and incorporate GEO database for user to compare their own dataset with those deposited in database. The comparison can be used to infer cooperative regulation and thus can be used to generate hypotheses. Several visualization functions are implemented to summarize the coverage of the peak experiment, average profile and heatmap of peaks binding to TSS regions, genomic annotation, distance to TSS, and overlap of peaks or genes.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioc">
        <div class="card">
          
          <a href="https://guangchuangyu.github.io/DOSE" title="" class="card-image hover-overlay"
             target="_blank">
            <img src="/img/dose_fig4.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/DOSE" target="_blank">DOSE</a></h4>
            <div class="card-desription">
              <p>This package implements five methods proposed by Resnik, Schlicker, Jiang, Lin and Wang respectively for measuring semantic similarities among DO terms and gene products. Enrichment analyses including hypergeometric model and gene set enrichment analysis are also implemented for discovering disease associations of high-throughput biological data.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioc">
        <div class="card">
          
          <a href="https://guangchuangyu.github.io/ReactomePA" title="" class="card-image hover-overlay"
             target="_blank">
            <img src="/img/reactome_GTEx.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/ReactomePA" target="_blank">ReactomePA</a></h4>
            <div class="card-desription">
              <p>Pathway analysis based on REACTOME pathway database. It implements enrichment analysis, gene set enrichment analysis and several functions for visualization.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioc">
        <div class="card">
          
          <a href="https://guangchuangyu.github.io/clusterProfiler" title="" class="card-image hover-overlay"
             target="_blank">
            <img src="/img/cp.png" alt="" class="img-responsive">
          </a>
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/clusterProfiler" target="_blank">clusterProfiler</a></h4>
            <div class="card-desription">
              <p>The clusterProfiler package implements methods to analyze and visualize functional profiles of genomic coordinates (supported by ChIPseeker), gene and gene clusters.</p>
            </div>
          </div>
        </div>
      </div>
      
      
      
      
      
      
      
      <div class="col-xs-12 col-sm-6 col-md-4 col-lg-6 project-item isotope-item bioc">
        <div class="card">
          
          <div class="card-text">
            <h4><a href="https://guangchuangyu.github.io/GOSemSim" target="_blank">GOSemSim</a></h4>
            <div class="card-desription">
              <p>The semantic comparisons of Gene Ontology (GO) annotations provide quantitative ways to compute similarities between genes and gene groups, and have became important basis for many bioinformatics analysis approaches. GOSemSim is an R package for semantic similarity computation among GO terms, sets of GO terms, gene products and gene clusters. GOSemSim implemented five methods proposed by Resnik, Schlicker, Jiang, Lin and Wang respectively.</p>
            </div>
          </div>
        </div>
      </div>
      

    </div>

    

  </div>
-->
  </div>

    </div>
  </section>

  
  
  <section id="working_cn" class="home-section">
    <div class="container">
      


<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>工作经历</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
<!--    <p>This is an example of using the <em>custom</em> widget to create your own homepage section.</p>
-->
<p>本人于2016年7月至2017年5月期间，在天津医科大学王举教授生物信息学实验室任研究助理。</p>
<!--
<ul>
<li>xx</li>
<li>yy</li>
</ul>
-->
  </div>
</div>

    </div>
  </section>

  
  
  <section id="contact_cn" class="home-section">
    <div class="container">
      




<div class="row">
    <div class="col-xs-12 col-md-4 section-heading">
        <h1>联系方式</h1>
        
    </div>
    <div class="col-xs-12 col-md-8">
        

        <ul class="fa-ul" itemscope>

            
            <li>
                <i class="fa-li fa fa-envelope fa-lg" aria-hidden="true"></i>
                <span id="person-email" itemprop="email"><a href="mailto:yanshihu@zju.edu.cn">yanshihu@zju.edu.cn</a></span>
            </li>
            

            

            

            

            

            
            <li>
                <i class="fa-li fa fa-map-marker fa-lg" aria-hidden="true"></i>
                <span id="person-address" itemprop="address">浙江大学</span>
            </li>
            

            

        </ul>
		<!--
		<iframe src="Baidu_map.html" width="700" height="550" frameborder="0" scrolling="no">
		
		</iframe>
		-->

  <body>
    <!--百度地图容器-->
    <div style="width:700px;height:550px;border:#ccc solid 1px;font-size:12px" id="map"></div>
  </body>
  <script type="text/javascript">
    //创建和初始化地图函数：
    function initMap(){
      createMap();//创建地图
      setMapEvent();//设置地图事件
      addMapControl();//向地图添加控件
      addMapOverlay();//向地图添加覆盖物
    }
    function createMap(){ 
      map = new BMap.Map("map"); 
      map.centerAndZoom(new BMap.Point(120.096545,30.302836),19);
    }
    function setMapEvent(){
      map.enableScrollWheelZoom();
      map.enableKeyboard();
      map.enableDragging();
      map.enableDoubleClickZoom()
    }
    function addClickHandler(target,window){
      target.addEventListener("click",function(){
        target.openInfoWindow(window);
      });
    }
    function addMapOverlay(){
      var markers = [
        {content:"浙大生科院生物信息学实验室",title:"",imageOffset: {width:-46,height:-21},position:{lat:30.302595,lng:120.096127}}
      ];
      for(var index = 0; index < markers.length; index++ ){
        var point = new BMap.Point(markers[index].position.lng,markers[index].position.lat);
        var marker = new BMap.Marker(point,{icon:new BMap.Icon("http://api.map.baidu.com/lbsapi/createmap/images/icon.png",new BMap.Size(20,25),{
          imageOffset: new BMap.Size(markers[index].imageOffset.width,markers[index].imageOffset.height)
        })});
        var label = new BMap.Label(markers[index].title,{offset: new BMap.Size(25,5)});
        var opts = {
          width: 200,
          title: markers[index].title,
          enableMessage: false
        };
        var infoWindow = new BMap.InfoWindow(markers[index].content,opts);
        marker.setLabel(label);
        addClickHandler(marker,infoWindow);
        map.addOverlay(marker);
      };
    }
    //向地图添加控件
    function addMapControl(){
      var scaleControl = new BMap.ScaleControl({anchor:BMAP_ANCHOR_BOTTOM_RIGHT});
      scaleControl.setUnit(BMAP_UNIT_IMPERIAL);
      map.addControl(scaleControl);
      var navControl = new BMap.NavigationControl({anchor:BMAP_ANCHOR_TOP_RIGHT,type:1});
      map.addControl(navControl);
      var overviewControl = new BMap.OverviewMapControl({anchor:BMAP_ANCHOR_BOTTOM_RIGHT,isOpen:false});
      map.addControl(overviewControl);
    }
    var map;
      initMap();
  </script>  
	</div>
</div>

    </div>
  </section>



<footer class="site-footer">
  <div style="width:320px;margin:auto;">
<script type="text/javascript" src="//ra.revolvermaps.com/0/0/7.js?i=0zldq3hfhxa&amp;m=0&amp;c=ff0000&amp;cr1=ffffff&amp;sx=0&amp;ds=20" async="async"></script>
</div>
  <div class="container">
    <p class="powered-by">

      &copy; 2018 胡言石 &middot; 

      技术支持：<a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    <script src="//cdnjs.cloudflare.com/ajax/libs/gsap/1.18.4/TweenMax.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/gsap/latest/plugins/ScrollToPlugin.min.js"></script>
    <script src="/js/jquery-1.12.3.min.js"></script>
    <script src="/js/bootstrap.min.js"></script>
    <script src="/js/isotope.pkgd.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.1/imagesloaded.pkgd.min.js"></script>
    <script src="/js/hugo-academic.js"></script>
    

    
    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
        ga('create', 'UA-77901140-1', 'auto');
        ga('send', 'pageview');

         
        var links = document.querySelectorAll('a');
        Array.prototype.map.call(links, function(item) {
            if (item.host != document.location.host) {
                item.addEventListener('click', function() {
                    var action = item.getAttribute('data-action') || 'follow';
                    ga('send', 'event', 'outbound', action, item.href);
                });
            }
        });
    </script>
    

    
    
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

